rfcs/README.md

```diff
@@ -0,0 +1,98 @@
+# oneDPL Design Documents/RFCs
+
+The Request for Comments (RFC) process intends to:
+
+- Propose and discuss ideas for the library evolution.
+- Communicate anticipated library-wide changes.
+- Collect feedback before implementation.
+- Increase transparency in decision-making.
+- Align different teams involved in the library development.
+
+Most modifications or new features will naturally start as a part of a 
+GitHub issue or discussion. Small changes do not require a formal RFC. 
+However, if the issue or discussion results in an idea for a significant 
+change or new feature that affects the library's public API or architecture, 
+we recommend creating a formal document that provides
+a detailed description and design of the proposed feature.
+
+This directory contains design documents (RFCs) approved, 
+implemented, or rejected for implementation.
+
+## RFC Directory Structure
+
+The design documents are stored in the `rfcs` directory, each placed 
+in a subdirectory under an `rfcs/<state>` directory. The possible states are:
+
+1. Proposed
+2. Experimental
+3. Supported
+4. Archived
+
+The `rfcs/proposed` directory contains RFCs for approved proposals
+that need to be implemented. These documents describe the overall design
+and API for the proposed functionality.
+
+The `rfcs/experimental` directory contains RFCs for experimental library features.
+In addition to the design, these documents describe the criteria for the described
+functionality to exit the experimental status.
+
+The `rfcs/supported` directory contains documents for the fully supported features,
+both implemented according to the library specification and provided as extensions.
```

Comment:
The specification is not needed for an extension.

However I do not think that "either... or" instead of "both... and" makes anything more explicit. A **single** fully supported feature can be **_either_** specified **_or_** an extension; but **a set** fully supported features consists of **_both_** specified features **_and_** extensions. Here I am speaking about the plurality of features.